Source/Core/Transforms.js

```diff
@@ -457,9 +459,9 @@ define([
      * are above the plane. Negative pitch angles are below the plane. Roll is the first rotation applied about the local east axis.
      *
      * @param {Cartesian3} origin The center point of the local reference frame.
-     * @param {Number} heading The heading angle in radians.
-     * @param {Number} pitch The pitch angle in radians.
-     * @param {Number} roll The roll angle in radians.
+     * @param {HeadingPitchRoll} headingPitchRoll A HeadingPitchRoll or the heading angle in radians.
+     * @param {Number?} pitch The pitch angle in radians if a HeadingPitchRoll object was not passed.
```

Comment:
Sorry if I wasn't clear, let's also remove `pitch` and `roll`.  Also generate the doc to doublecheck that it looks OK; it should be fine since I believe the tool will only look at these comments, not the function signature.